src/styles.scss

```diff
@@ -2080,34 +2080,34 @@ input {
 
 @keyframes pulse-red {
   0% {
-    transform: scale(0.95);
+    // transform: scale(0.95);
```

Comment:
Perhaps just remove these if they're not being used

Reply:
Yes